crm_claim_type/README.rst

```diff
@@ -0,0 +1,60 @@
+CRM Claim Types
+==============
+
+This module was written to extend crm_claim module features.
+It gives a section to define a set of claim types.
```

Comment:
Add the specific use cas it solves.....

The expresion "This module was written" should not start the documentation of any modules "Iti onvious that it is a module and we see letters there eveybody know it was written jejeje".

Then it should say.

Include a way to clasify claims adding the concept of "type".

IMAGE WITHOUT THIS MODULE:

IMAGE WITH THIS MODULE: